tests/test_b4uleave.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Dict
+from platzky.platzky import create_app_from_config, Config
+
+
+def test_that_plugin_loads_b4uleave():
+
+    data_with_plugin: Dict[str, Any] = {
+        "APP_NAME": "testingApp",
+        "SECRET_KEY": "secret",
+        "USE_WWW": False,
+        "BLOG_PREFIX": "/",
+        "TRANSLATION_DIRECTORIES": ["/some/fake/dir"],
+        "DB": {
+            "TYPE": "json",
+            "DATA": {
+                "site_content": {"pages": []},
+                "plugins": [
+                    {
+                        "name": "b4uleave",
+                        "config": {
+                            "message": "Your custom message goes here"
+                        },
+                    }
+                ],
+            },
+        },
+    }
+
+    b4uleave_function = "B4ULeave-ModalWindow"
+
+    config_with_plugin = Config.model_validate(data_with_plugin)
+
+    app_with_plugin = create_app_from_config(config_with_plugin)
+
+    response = app_with_plugin.test_client().get("/")
+    
+    assert response.status_code == 200
+    decoded_response = response.data.decode()
+    assert (
+        b4uleave_function in decoded_response
```

Comment:
lets run event which is called on leaving site.

Reply:
and then check in if .B4ULeave-Content there is message "your custom message goes here"